File: cpp/10_input.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string name;
    cout << "What is your name? ";
    getline(cin, name);
    cout << "Hello " << name << "!" << endl;

    int age;
    cout << "How old are you? ";
    cin >> age;
    cout << "You are " << age << " years old." << endl;
    cout << "Next year you will be " << age + 1 << endl;

    return 0;
}

File: java/10_input.java
import java.util.Scanner;

public class Input {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("What is your name? ");
        String name = sc.nextLine();
        System.out.println("Hello " + name + "!");

        System.out.print("How old are you? ");
        int age = sc.nextInt();
        System.out.println("You are " + age + " years old.");
        System.out.println("Next year you will be " + (age + 1));

        sc.close();
    }
}
